network/alsp/misbehavior.go

```diff
@@ -0,0 +1,31 @@
+package alsp
+
+import "github.com/onflow/flow-go/network"
+
+const (
+	// StaleMessage is a misbehavior that is reported when an engine receives a message that is deemed stale based on the
+	// local view of the engine. The decision to consider a message stale is up to the engine.
+	StaleMessage network.Misbehavior = "misbehavior-stale-message"
+
+	// HeavyRequest is a misbehavior that is reported when an engine receives a request that takes an unreasonable amount
+	// of resources by the engine to process, e.g., a request for a large number of blocks. The decision to consider a
+	// request heavy is up to the engine.
+	HeavyRequest network.Misbehavior = "misbehavior-heavy-request"
```

Comment:
```suggestion
	// ResourceIntensiveRequest is a misbehavior that is reported when an engine receives a request that takes an unreasonable amount
	// of resources by the engine to process, e.g., a request for a large number of blocks. The decision to consider a
	// request heavy is up to the engine.
	ResourceIntensiveRequest network.Misbehavior = "misbehavior-resource-intensive-request"
```

Reply:
Added `InvalidRequest` in https://github.com/onflow/flow-go/pull/4099/commits/83589814c548fa56ee0f38bc69f4dac14b74762e. 

These will be for logging and updating the penalty records. However, the decision to tune the penalty value is done through the [engine](https://github.com/onflow/flow-go/blob/6c745ed04a74497c95f7195c637a52fe54470fb8/network/alsp/report.go#L69). If no penalty is specified, a default one is set.